### PR TITLE
[#801] Redirect to login page in case of token refresh failure

### PR DESCRIPTION
**Pull request by jonase**
- [ ] Update release notes
- [ ] Test plan | Unit test | Integration test
- [ ] Copyright header
- [ ] Code formatting
- [ ] Documentation

### CODE REVIEW

File: client/src/auth.js [hunk 1]
```diff
@@ -9,10 +9,13 @@ export function token() {
...
   }
 
-  return new Promise((resolve, reject) =>
+  return new Promise(resolve =>
     keycloak.updateToken()
       .success(() => resolve(keycloak.token))
-      .error(err => reject(err))
+      .error(() => {
+        // Refreshing will redirect to login page
+        window.location.reload(true);
```
Comment (iperdomo) [inline client/src/auth.js]: Why not use `keycloak.login()` ?

Comment (jonase) [inline client/src/auth.js]: Good point. Changed to `keycloak.login` instead.

**Comment by iperdomo** [comment]
Will try it locally before merging

**Comment by iperdomo** [comment]
👍

**Pull request merged**